src/gui/transferlistwidget.cpp

```diff
@@ -147,12 +147,12 @@ TransferListWidget::TransferListWidget(QWidget *parent, MainWindow *main_window)
 
     // Listen for list events
     connect(this, SIGNAL(doubleClicked(QModelIndex)), this, SLOT(torrentDoubleClicked(QModelIndex)));
-    connect(this, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(displayListMenu(const QPoint &)));
+    connect(this, SIGNAL(customContextMenuRequested(const QPoint&)), this, SLOT(displayListMenu(const QPoint&)));
     header()->setContextMenuPolicy(Qt::CustomContextMenu);
-    connect(header(), SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(displayDLHoSMenu(const QPoint &)));
-    connect(header(), SIGNAL(sectionMoved(int, int, int)), this, SLOT(saveSettings()));
-    connect(header(), SIGNAL(sectionResized(int, int, int)), this, SLOT(saveSettings()));
-    connect(header(), SIGNAL(sortIndicatorChanged(int, Qt::SortOrder)), this, SLOT(saveSettings()));
+    connect(header(), SIGNAL(customContextMenuRequested(const QPoint&)), this, SLOT(displayDLHoSMenu(const QPoint&)));
+    connect(header(), SIGNAL(sectionMoved(int,int,int)), this, SLOT(saveSettings()));
```

Comment:
I've checked uncrustify, and this action in intentional and unconditional. The comments in the source code say that they change such lines closer to the normalised form. The performance of the code being changed could be the only reason to that.